pandas/io/sas/meson.build

```diff
@@ -0,0 +1,23 @@
+py.extension_module(
+    '_sas',
+    ['sas.pyx'],
+    include_directories: [inc_np],
+    dependencies: [py_dep],
+    # The file is named sas.pyx but we want the
+    # extension module to be named _sas
+    cython_args: ['--module-name=pandas.io.sas._sas'],
```

Comment:
Probably easier just to rename the file to _sas.pyx. Can also be done in a pre cursor - no real reason why this is this way today